ui/src/components/advent/AdventCard.js

```diff
@@ -0,0 +1,238 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalCloseButton,
+  ModalContent,
+  Image,
+  useDisclosure,
+  ModalHeader,
+  Box,
+  AspectRatio,
+  Center,
+  Text,
+  VStack,
+  HStack,
+  Link,
+  ModalFooter,
+  ChakraProvider,
+} from '@chakra-ui/react';
+
+import adventTheme from './adventTheme';
+import ReactMarkdown from 'react-markdown';
+import ChakraUIRenderer from 'chakra-ui-markdown-renderer';
+import rehypeRaw from 'rehype-raw';
```

Comment:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/24607458/204101494-274dfb41-fe16-40fb-a281-5e25870ab7d1.png">

seeing this issue even after yarn install,  when the packages are needed to be installed, the packge.json should also be commited,